app/dashboard/templates/dashboard/users.html

```diff
@@ -251,7 +251,18 @@ <h6 class="font-weight-bold mb-3">Invite User to Bounty</h6>
     {% include 'shared/footer.html' %}
     {% include 'shared/messages.html' %}
 
+    {% compress js %}
+    <script src="{% static "v2/js/lib/popper.min.js" %}"></script>
+    <script src="{% static "v2/js/lib/bootstrap.min.js" %}" crossorigin="anonymous"></script>
+    {% endcompress %}  
+    
+    
     <script>
+      let bootstrapTooltip = $.fn.tooltip.noConflict()
+      $.fn.runTooltip = bootstrapTooltip;
```

Comment:
also declared in footer_scripts

Reply:
1eb3739d21bc4c23760d2eae6fdcf96cc8d5bdab